Gordon360/ApiControllers/NewsController.cs

```diff
@@ -151,6 +150,27 @@ public IHttpActionResult Post([FromBody] StudentNews newsItem)
             return Created("News", result);
         }
 
-
+        /// <summary>Deletes a news item from the database</summary>
+        /// <param name="newsID">The id of the news item to delete</param>
+        /// <returns>The deleted news item</returns>
+        /// <remarks>The news item must be authored by the user and must not be expired</remarks>
+        [HttpDelete]
+        [Route("{newsID}")]
+        [StateYourBusiness(operation = Operation.DELETE, resource = Resource.NEWS)]
+        // Private route to authenticated authors of the news entity
+        public IHttpActionResult Delete(int newsID)
+        {
+            // StateYourBusiness verfies that user is authenticated
+            // Delete permission should be allowed only to authors of the news item
+            // News item must not have already expired
+            var result = _newsService.DeleteNews(newsID);
+            // Shouldn't be necessary
+            if(result == null)
+            {
+                return NotFound();
+            }
+            return Ok(result);
+        }
+        
     }
 }
```

Comment:
I'm noticing on a few of these files that GitHub is marking that there is no newline character at the end of the file. Some people argue that ending in a newline should be convention while others ignore it. What do you think we should do?

Reply:
Sounds good. I'll keep an eye out on any documents I change going forward as well.